.github/workflows/test.yml

```diff
@@ -1,12 +1,86 @@
 name: Tests
-on: push
+on: [pull_request, push]
 jobs:
-  tests:
+  linting:
+    name: Linting
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Setup Python 3.8
+        uses: actions/setup-python@v1
+        with:
+          python-version: "3.8"
+          architecture: x64
+
```

Comment:
```suggestion
```
It will probably be fastest to just run the linting tools on the default latest and greatest.

Reply:
We don't support the latest and greatest. Tensor flow doesn't support python >= 3.8 

I feel like It's best to test wit the version you support even if I agree its very unlikely to make a difference.